app/src/main/java/beforeget/feature/main/MainAdapter.kt

```diff
@@ -0,0 +1,46 @@
+package beforeget.data
+
+import android.content.Intent
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.core.content.ContextCompat
+import androidx.core.content.ContextCompat.startActivity
+import androidx.recyclerview.widget.RecyclerView
+import beforeget.feature.MyRecordActivity
+import com.example.beforeget.databinding.ItemMediaListBinding
+
+
+class MainAdapter : RecyclerView.Adapter<MainAdapter.MainViewHolder>() {
+
+    val mediaList = mutableListOf<MainData>()
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MainViewHolder {
+        val binding = ItemMediaListBinding.inflate(
+            LayoutInflater.from(parent.context), parent, false
+        )
+        return MainViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: MainViewHolder, position: Int) {
+        holder.onBind(mediaList[position])
+    }
+
+    override fun getItemCount(): Int = mediaList.size
+
+    class MainViewHolder(private val binding: ItemMediaListBinding) :
+        RecyclerView.ViewHolder(binding.root) {
```

Comment:
```suggestion
    class MainViewHolder(
        private val binding: ItemMediaListBinding
    ) : RecyclerView.ViewHolder(binding.root) {
```
이렇게 줄 바꿈 처리하면 보다 가독성있게 처리할 수 있습니다~